langkit/transformer.py

```diff
@@ -28,7 +33,6 @@ def __init__(
         self,
         transformer_name: Optional[str],
         custom_encoder: Optional[Callable[[List[str]], Any]],
-        veto_cuda: bool = False,
```

Comment:
Do we need to remove this? What's the context?

Reply:
put this back in